.changeset/calm-rooms-build.md

```diff
@@ -0,0 +1,14 @@
+---
+'sku': patch
+'@sku-lib/create': patch
+---
+
+Remove `@sku-lib/vitest` as a peer dependency.
+
+`@sku-lib/vitest` is now included in the core `sku` package. If you use `testRunner: 'vitest'` in your sku config, simply add `vitest` as a dev dependency:
```

Comment:
Realised it might already be installed in most cases (assuming the consumer has followed the migration docs), so this is more of a reminder than a suggestion.

```suggestion
`@sku-lib/vitest` is now included in the core `sku` package. If you use `testRunner: 'vitest'` in your sku config, simply add `vitest` as a dev dependency if it is not already present:
```